docs/src/modules/components/DemoToolbar.js

```diff
@@ -363,6 +364,71 @@ export default function DemoToolbar(props) {
     isFocusableControl,
   });
 
+  const devMenuItems = [];
+  if (process.env.STAGING === true) {
+    /* eslint-disable material-ui/no-hardcoded-labels -- staging only */
+    // eslint-disable-next-line react-hooks/rules-of-hooks -- process.env.STAGING never changes
+    const router = useRouter();
+
+    const defaultReviewID = process.env.GIT_REVIEW_ID ?? '20000';
+    devMenuItems.push(
+      <MenuItem
+        key="link-next"
+        data-ga-event-category="demo"
+        data-ga-event-label={demoOptions.demo}
+        data-ga-event-action="link-next"
+        component="a"
+        href={`https://deploy-preview-${defaultReviewID}--${process.env.NETLIFY_SITE_NAME}.netlify.app${router.route}/#${demoName}`}
+        target="_blank"
+        rel="noopener nofollow"
+        onClick={handleMoreClose}
+      >
+        demo on PR #{defaultReviewID}
+      </MenuItem>,
+      <MenuItem
+        key="link-next"
```

Comment:
We have twice the same key, is it expected?

Reply:
Fixed in [`af691bf` (#26065)](https://github.com/mui-org/material-ui/pull/26065/commits/af691bf742b86a9a25c327af563fecd172b22612)